src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsList.jsx

```diff
@@ -1,12 +1,17 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
 import ViewDependentsListItem from '../ViewDependentsList/ViewDependentsListItem';
 
 class ViewDependentsList extends Component {
   render() {
     let mainContent;
 
-    if (this.props.dependents) {
+    if (this.props.loading) {
+      mainContent = (
+        <LoadingIndicator message="Loading your dependents" setFocus />
+      );
+    } else if (this.props.dependents.length > 0) {
       mainContent = this.props.dependents.map((dependent, index) => (
         <ViewDependentsListItem
           key={index}
```

Comment:
This could be {...dependent} if you wanted to. Weird, I could have sworn this was merged in by another PR.